src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/totrinnskontroll/SendTilBeslutterSteg.kt

```diff
@@ -105,9 +104,12 @@ class SendTilBeslutterSteg(
         }
     }
 
-    private fun validerAtDetFinnesOppgave(saksbehandling: Saksbehandling) {
-        feilHvis(oppgaveService.hentBehandleSakOppgaveSomIkkeErFerdigstilt(saksbehandling.id) == null) {
-            "Oppgaven for behandlingen er ikke tilgjengelig. Vennligst vent og prøv igjen om litt."
+    private fun validerOppgaver(saksbehandling: Saksbehandling) {
+        brukerfeilHvis(oppgaveService.hentBehandleSakOppgaveSomIkkeErFerdigstilt(saksbehandling.id) == null) {
+            "Oppgaven for behandlingen er ikke tilgjengelig. Prøv igjen om litt."
+        }
+        brukerfeilHvis(oppgaveService.hentOppgaveSomIkkeErFerdigstilt(saksbehandling.id, Oppgavetype.GodkjenneVedtak) != null) {
+            "Systemet har ikke rukket å ferdigstille godkjenne vedtak-oppgaven. Prøv igjen om litt."
```

Comment:
Gir denne feilmeldingen mening for bruker? 🤔 Evt. si noe om at det er fordi den ble angret

Reply:
Er det egentlig mulig å få denne feilmeldingen med mindre man sender til beslutter veldig kjapt etter angring eller systemet har hengt seg opp?

Reply:
Kun hvis vi har gjort noe feil